Libraries/LibWeb/DOM/Document.cpp

```diff
@@ -4454,6 +4454,9 @@ bool Document::is_allowed_to_use_feature(PolicyControlledFeature feature) const
     case PolicyControlledFeature::FocusWithoutUserActivation:
         // FIXME: Implement allowlist for this.
         return true;
+    case PolicyControlledFeature::Fullscreen:
+        // FIXME: Implement the permissions policy specification
+        return true;
```

Comment:
Commit message: why is the policy check "stupid"? If there's a good reason to call it that then add that reasoning to the commit message, if it's just subjective then leave it out please.